es/es-server/src/main/java/org/elasticsearch/index/analysis/SynonymTokenFilterFactory.java

```diff
@@ -54,34 +54,14 @@ public SynonymTokenFilterFactory(IndexSettings indexSettings, Environment env, A
                                       String name, Settings settings) throws IOException {
         super(indexSettings, name, settings);
         this.settings = settings;
-
-        this.ignoreCase =
-            settings.getAsBooleanLenientForPreEs6Indices(indexSettings.getIndexVersionCreated(), "ignore_case", false, deprecationLogger);
-        if (indexSettings.getIndexVersionCreated().onOrAfter(Version.ES_V_6_1_4) && settings.get("ignore_case") != null) {
+        this.ignoreCase = settings.getAsBoolean("ignore_case", false);
```

Comment:
afaik the instance var is unused now and can be removed

Reply:
It is used in the `SynonymGraphTokenFilterFactory` sub-class

Reply:
ah right